docs/src/man/themes.md

```diff
@@ -82,13 +95,51 @@ srand(12345)
 ```
 
 ```@example 1
-plot(x=rand(10), y=rand(10),
-     Theme(panel_fill=colorant"black", default_color=colorant"orange"))
+
+dark_panel = Theme(
+    panel_fill=colorant"black",
+    default_color=colorant"orange"
+)
+
+plot(x=rand(10), y=rand(10), dark_panel)
+
 ```
 
 Setting the font to Computer Modern to create a LaTeX-like look, and choosing a font size:
 
 ```@example 1
+Gadfly.push_theme(dark_panel)
+
 plot(x=rand(10), y=rand(10),
-     Theme(major_label_font="CMU Serif",minor_label_font="CMU Serif",major_label_font_size=16pt,minor_label_font_size=14pt))
+     style(major_label_font="CMU Serif",minor_label_font="CMU Serif",
+           major_label_font_size=16pt,minor_label_font_size=14pt))
+
+# can plot more plots here...
+
+Gadfly.pop_theme()
+```
+
+Same effect can be had with `with_theme`
+
+```@example 1
+Gadfly.with_theme(dark_panel) do
+
+  plot(x=rand(10), y=rand(10),
+       style(major_label_font="CMU Serif",minor_label_font="CMU Serif",
+             major_label_font_size=16pt,minor_label_font_size=14pt))
+end
```

Comment:
As in, you don't want it to plot? You can append ` # hide` (spaces are important) to the end of any line you don't want displayed. If you have multiple lines then a `@setup` block might be useful. See https://juliadocs.github.io/Documenter.jl/latest/man/syntax.html#@example-block-1